pkgs/by-name/ft/fte/package.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  unzip,
+  perl,
+  libX11,
+  libXpm,
+  gpm,
+  ncurses,
+  slang,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "fte";
+  version = "0.50.02";
+
+  ftesrc = fetchurl {
+    url = "mirror://sourceforge/fte/fte-20110708-src.zip";
+    hash = "sha256-1jEcVC0/DyiQpUpmHDtnIo4nuJS0Fk6frynwFPJUSZ4=";
+  };
+
+  ftecommon = fetchurl {
+    url = "mirror://sourceforge/fte/fte-20110708-common.zip";
+    hash = "sha256-WEEVeLMZWHZfQtK/Kbeu3Z+RaVXCwZyWkJocA+Akavc=";
+  };
```

Comment:
Where does 0.50.02 come from?

Reply:
within the `fte-20110708-common.zip`, there is a `NEWS` file:

```
$ cat NEWS
Revision history
-----------------------------------------------------------------------
0.50.2b6 -- <fill month here> 2011
  + Allow \r at the end-of-line when looking for the end-of-heredoc in
    Perl hilighting mode to make it easier to read files created on windows
    and transfered to unix incorrectly.
  + Add ShowTildeFilesInDirList global config option to allow hiding
    of *~ in the directory list (most of the time, I don't want this)
    default is true.
  ! Allow open-brace on same line as function name in shell mode
  + Added permission flags to the directory listings on unix to show rwx
    at a glance - makes it easier to debug certain issues, know if you
    can write to the file, etc.
  ! Compilation fixes for Windows compilers
  + Allow JAVA colorizer to be used for javascript files (.js)
  - Ongoing conversion of return codes to return consistently
    1 for success or 0 for failure
  ! Colorizing man pages again (using MAN_KEEP_FORMATTING)
  + adding 2nd FD for WaitFdPipeEvent() so used in con_linux.cpp

0.50.2b5 -- March 2010
  + Allowed -m flag to take lower-cased version of mode names
  ! .t files are perl test files, added.
...
```